torchqdynamics/utils/utils.py

```diff
@@ -22,95 +22,93 @@
 ]
 
 
-def is_ket(x: Tensor) -> Tensor:
-    """Check if a tensor is in state vector format."""
+def is_ket(x: Tensor) -> bool:
+    """Returns True if a tensor is in the format of a state vector, i.e. if its last
+    dimension is 1."""
```

Comment:
So in the end should this match the other functions and also add a Parameters and Returns section? At least the Parameters one I think is important, even if its just to write "the input tensor". Right now I feel like it's too short haha. Sorry I'm annoying >.<